### ISSUE TITLE
Implement keyboard shortcuts for operations in grid editor

### ISSUE THREAD

**Issue by sdruskat** [hexatomic/hexatomic#243]
**Is your feature request related to a problem? Please describe.**

Keyboard shortcuts should be available at least for all opreations in the grid editor which otherwise cannot be done by keyboard only, including, but not limited to:

- [ ] Create and annotate new span

**Describe the solution you'd like**

- Keyboards shortcuts for grid editor operations are implemented
- Menmonics are listed in the context menu
- Mnemonics follow an agreed-upon "keyspace", e.g., all in-.editor mnemonics across Hexatomic are <kbd>Alt</kbd> + `letter`shortcuts.

**Additional context**

Issue only has the grid editor in scope, but should perhaps be extended to cover all existing functionality?
